### PR TITLE
Update building-a-business-partnership-as-a-thriving-ecosystem.md

### PR DESCRIPTION
**Pull request by laurenwigmore**
bulleted list and image link updated

### CODE REVIEW

**Comment by rufuspollock** [comment]
@laurenwigmore i see errors like:

```
18:38:24.119 | └── Encountered unexpected errors while processing of 1 documents. This is possibly a bug in Contentlayer. Please open an issue.
-- | --
18:38:24.120 |  
18:38:24.120 | • "blog/building-a-business-partnership-as-a-thriving-ecosystem.md": UnexpectedMDXError: Error: Build failed with 1 error:
18:38:24.120 | _mdx_bundler_entry_point-0cd84553-8f23-4cba-ab2c-1adbd03fab93.mdx:19:99: ERROR: [plugin: @mdx-js/esbuild] Unexpected closing slash `/` in tag, expected an open tag first
```

It's probably some html in the file where you don't have tags right ...

File: content/blog/building-a-business-partnership-as-a-thriving-ecosystem.md [hunk 1]
```diff
@@ -5,23 +5,23 @@ categories:
   - community-call
 authors:
   - moon-immisch
-image: /assets/images/elizabeth&myka
+image: /assets/images/elizabeth&myka.jpg
 Decription: We’ve all heard the horror stories of business partnerships gone wrong, but Elizabeth and Myka have learned how to create their company and partnership like a thriving community and ecosystem.
 ---
-**[Register for the call on Thursday 7 December at 1930CET]**(**[https://us02web.zoom.us/meeting/register/tZcpcu-oqTkiGNIO3Xs7xcUQqMQ48hqhHQ9M](https://us02web.zoom.us/meeting/register/tZcpcu-oqTkiGNIO3Xs7xcUQqMQ48hqhHQ9M)**)
+**[Register for the call on Thursday 7 December at 1930CET](**[https://us02web.zoom.us/meeting/register/tZcpcu-oqTkiGNIO3Xs7xcUQqMQ48hqhHQ9M](https://us02web.zoom.us/meeting/register/tZcpcu-oqTkiGNIO3Xs7xcUQqMQ48hqhHQ9M)**)**
```
Comment (rufuspollock) [inline content/blog/building-a-business-partnership-as-a-thriving-ecosystem.md:11]: This is weird - you have the link twice ...

Comment (rufuspollock) [reply content/blog/building-a-business-partnership-as-a-thriving-ecosystem.md:11]: I think this is prob the issue

File: content/blog/building-a-business-partnership-as-a-thriving-ecosystem.md [hunk 2]
```diff
@@ -5,23 +5,23 @@ categories:
   - community-call
 authors:
   - moon-immisch
-image: /assets/images/elizabeth&myka
+image: /assets/images/elizabeth&myka.jpg
```
Comment (rufuspollock) [inline content/blog/building-a-business-partnership-as-a-thriving-ecosystem.md:8]: & in urls is going to be tricky ... i would suggest correctly the `&` to something else like and.

**Comment by laurenwigmore** [comment]
> @laurenwigmore i see errors like:
> 
> ```
> 18:38:24.119 | └── Encountered unexpected errors while processing of 1 documents. This is possibly a bug in Contentlayer. Please open an issue.
> -- | --
> 18:38:24.120 |  
> 18:38:24.120 | • "blog/building-a-business-partnership-as-a-thriving-ecosystem.md": UnexpectedMDXError: Error: Build failed with 1 error:
> 18:38:24.120 | _mdx_bundler_entry_point-0cd84553-8f23-4cba-ab2c-1adbd03fab93.mdx:19:99: ERROR: [plugin: @mdx-js/esbuild] Unexpected closing slash `/` in tag, expected an open tag first
> ```
> 
> It's probably some html in the file where you don't have tags right ...

OK I'll re look at it

**Comment by laurenwigmore** [comment]
@rufuspollock on Github desktop I have 2 files that are showing up as conflicted. I took these across to the other branch earlier unintentionally, and then brought them back to the main when I realised, is it to do with these? And am I OK to discard changes?
image

**Pull request merged**